features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml.j2

```diff
@@ -303,6 +303,7 @@
                 <Scope name="internal_authenticator_view"/>
                 <Scope name="internal_governance_view"/>
                 <Scope name="internal_extensions_view"/>
+                <Scope name="internal_config_mgt_view"/>
```

Comment:
To be consistent shall we add these changes to api-resource-collection.xml file as well?